rwt_image_view/launch/example.launch

```diff
@@ -1,6 +1,10 @@
 <launch>
-  <node pkg="roswww" type="webserver.py" name="roswww">
-  </node>
+  <arg name="launch_roswww" default="true" />
+  <include file="$(find roswww)/launch/roswww.launch"
+           if="$(arg launch_roswww)">
```

Comment:
I'm wondering why you need this line. Is there a situation where you don't need to run `roswww` while using these launch?